packages/client/auth-core/src/components/ActionModal.tsx

```diff
@@ -0,0 +1,56 @@
+import { Dialog, Transition } from "@headlessui/react";
+import { Fragment, ReactNode } from "react";
+
+export type ActionModalProps = {
+    show: boolean;
+    onClose?: () => void;
+};
+
+export default function ActionModal({
+    show,
+    onClose = () => {},
+    children,
+}: ActionModalProps & { children: ReactNode }) {
+    // WAL-2574 - because of a recent change from another sdk in @headlessui/react, this file started complaining about the types
+    return (
+        <Transition.Root show={show} as={Fragment}>
+            <Dialog
+                as="div"
+                className="fixed inset-0 z-20 flex items-center justify-center overflow-y-auto"
```

Comment:
pretty sure these tailwind classes will not work in multiple cases:
1.) if the dev isnt using tailwind
2.) if they are: normally tailwind compiles to the smallest possible css file, so even if the dev is using tailwind these specific classes might not be in the outputted css file

Reply:
Perfect, thanks for highlighting it!

Reply:
updated to vanilla css